screen/.eslintrc.json

```diff
@@ -0,0 +1,21 @@
+{
+    "env": {
+        "browser": true,
+        "commonjs": true,
+        "es2020": true,
+        "node": true
+    },
+    "extends": [
+        "standard"
+    ],
+    "parserOptions": {
+        "ecmaVersion": 11
+    },
+    "rules": {
+        "quotes": ["error", "double"],
+        "indent": ["error", 4, {"SwitchCase": 1}],
```

Comment:
I personally prefer 2 spaces indentation for Javascript, so if we want to be consistent I will argue for that.

I don't think it's that important to have the same style in the different projects, though, so if you prefer 4 spaces I'm fine with having 4 here and 2 in UI.

Reply:
I'm a big fanboy of 4 spaces so would much want that 8-D